pkg/expr/nodes.go

```diff
@@ -261,11 +263,8 @@ func executeDSNodesGrouped(ctx context.Context, now time.Time, vars mathexp.Vars
 				if e != nil {
 					responseType = "error"
 					respStatus = "failure"
-					span.AddEvents([]string{"error", "message"},
-						[]tracing.EventValue{
-							{Str: fmt.Sprintf("%v", err)},
-							{Str: "failed to query data source"},
-						})
+					span.SetStatus(codes.Error, "failed to query data source")
```

Comment:
@grafana/observability-metrics is it bad to set the span status to error here?

Reply:
I think this is okay